src/l500/l500-device.cpp

```diff
@@ -126,6 +126,18 @@ namespace librealsense
         register_info(RS2_CAMERA_INFO_PRODUCT_LINE, "L500");
         register_info(RS2_CAMERA_INFO_CAMERA_LOCKED, _is_locked ? "YES" : "NO");
 
+        // If FW supportes the SET_AGE command, we update the age of the device in weeks to aid projection of aging
+        if ((_fw_version >= firmware_version("1.5.1.4"))) {
```

Comment:
Needs to be changed -- please check with Dubi which version should support this

Reply:
And formatting ({} on own lines)

Reply:
Dubi says they still don't know what version will support the new command. Perhaps we should close the PR and wait for them, what do you think?

Reply:
Make it 1.5.3 for now